Source/GitReleaseManager/GitHubProvider.cs

```diff
@@ -294,6 +294,20 @@ public async Task CloseMilestone(string owner, string repository, string milesto
             }
         }
 
+        public async Task OpenMilestone(string owner, string repository, string milestoneTitle)
+        {
+            var milestoneClient = _gitHubClient.Issue.Milestone;
+            var closedMilestones = await milestoneClient.GetAllForRepository(owner, repository, new MilestoneRequest { State = ItemStateFilter.Closed }).ConfigureAwait(false);
```

Comment:
This could use the `GetIssuesFromMilestoneAsync` method instead of calling the client directly.

Reply:
Sorry, misread the method.